src/components/HFUI/HFUI.js

```diff
@@ -18,26 +17,52 @@ import NotificationsSidebar from '../NotificationsSidebar'
 
 import Routes from '../../constants/routes'
 import { isElectronApp } from '../../redux/config'
+import MIN_SAFE_WIDTH from '../../constants/MIN_SAFE_WIDTH'
+import BestExperienceMessageModal from '../BestExperienceMessageModal'
 
 import './style.css'
 
+const HF_UI_WEB_SHOW_BEST_EXPERIENCE_MODAL = 'HF_UI_WEB_SHOW_BEST_EXPERIENCE_MODAL'
+
 const HFUI = ({
   authToken, getSettings, notificationsVisible, getFavoritePairs, currentMode, GAPageview,
   currentPage, onUnload, subscribeAllTickers, shouldShowAOPauseModalState, settingsShowAlgoPauseInfo,
 }) => {
-  function unloadHandler() {
```

Comment:
`unloadHandler` has to be a traditional `function`, not an arrow one (as well as the `onElectronAppClose`).
that's because `removeEventListener` needs a reference to the function which was used for `addEventListener`, this cannot be achieved with anonymous arrow functions